tests/test_transpile_self.py

```diff
@@ -246,3 +257,29 @@ def test_julia_recursive(self):
             format_errors,
             failures,
         )
+
+    def test_vlang_recursive(self):
+        settings = self.SETTINGS["vlang"]
+        transpiler_module = ROOT_DIR / "pyv"
+
+        assert_some_failures(
+            *_process_dir(
+                settings,
+                transpiler_module,
+                OUT_DIR,
+                False,
+                _suppress_exceptions=(AstNotImplementedError, AstUnrecognisedBinOp),
+            ),
+            expected_success={"clike.py"},
+        )
+        assert_counts(
+            *_process_dir(
+                settings,
+                PY2MANY_MODULE,
+                OUT_DIR,
+                False,
+                _suppress_exceptions=(AstNotImplementedError, AstUnrecognisedBinOp),
+            ),
+            format_error_count=2,
+            failure_count=12,
```

Comment:
ping @div72 , it would be good to get these errors down to 0; probably only a few bits of syntax needed.

Reply:
I'll take a look.

Reply:
Created https://github.com/py2many/py2many/issues/474 and gave it a `V` label